evidence/claravanstaden/0003-claravanstaden-2025-01-retention-rank-1.md

```diff
@@ -0,0 +1,49 @@
+# Argument-0003: Retention at/Promotion to Rank I
```

Comment:
Which one? Or do you want to use it for both in case of bump?

Reply:
Updated to retention: https://github.com/polkadot-fellows/Evaluations/pull/138/commits/ab3aa2301b887f1948a31735c2e316cb6958033d